apps/builder/app/builder/builder.tsx

```diff
@@ -74,7 +75,7 @@ const useNavigatorLayout = () => {
   // We need to render the detached state only once the setting was actually loaded from local storage.
   // Otherwise we may show the detached state because its the default and then hide it immediately.
   const [clientSettings, _, isLoaded] = useClientSettings();
-  return isLoaded ? clientSettings.navigatorLayout : "docked";
+  return isLoaded ? clientSettings.navigatorLayout : "undocked";
```

Comment:
please don't change this behavior by default, we want users to have navigator undocked by default because that's the most common need, in reality people use it all the time

Reply:
undocked.
<img width="987" alt="image" src="https://user-images.githubusercontent.com/5077042/220749317-6b143a8e-f9c3-4924-93bd-963722891a77.png">

docked
<img width="981" alt="image" src="https://user-images.githubusercontent.com/5077042/220749566-fc5495c4-f2dc-415d-89e6-458d11371704.png">

I see the same picture but in the case of "undocked" there is no layout trash
So I'm not getting why I can't change it

Reply:
ohh, I thought you changed the other way around, but also, did you try to set it do docked on the client and reload? now it should be loading as undocked with your change and then jumping to docked

Reply:
probably we should just add a transition for this so that this doesn't feel like a jump but rather as a nice, wanted transition

Reply:
I think we should move docked/undocked into searchParams. Or switch on client rendering. Transitions doesn't look good.

Reply:
searchParams won't solve it for permanently setting it, I think we could save it in the db, then ssr knows the setting

Reply:
but this problem is just one case, we will have more later, so we need a way to allow local settings based on local storage I think